contrib/babelfishpg_tds/src/backend/tds/tdsresponse.c

```diff
@@ -517,6 +541,108 @@ resolve_numeric_typmod_outer_var(Plan *plan, AttrNumber attno)
 	return resolve_numeric_typmod_from_exp(outerplan, (Node *)tle->expr);
 }
 
+static bool is_tsql_bit_numeric(Oid oid)
+{
+	if (tsql_bit_numeric_oid == InvalidOid)
+	{
+		Oid			nspoid;
+		Oid			typoid;
+		Oid 		funcargtypes[1];
+
+		nspoid = get_namespace_oid("sys", true);
+		if (nspoid == InvalidOid)
+			return InvalidOid;
+
+		typoid = GetSysCacheOid2(TYPENAMENSP, Anum_pg_type_oid, CStringGetDatum("bit"), ObjectIdGetDatum(nspoid));
+
+		funcargtypes[0] = typoid;
+		tsql_bit_numeric_oid = LookupFuncName(list_make2(makeString("sys"), makeString("bit2numeric")), 1, funcargtypes, true);
```

Comment:
missing_ok is passed as true which means it may return invalid oid. We should detect that and throw an error. IMO, we should pass missing-ok = false